New developer boards section and description for the HiHope SCDAY200 board (#15)
eclipse-oniro4openharmony/eclipse-oniro4openharmony.github.io: dev-boards.md

---
layout: default
title: Developer Boards
nav_order: 2
has_children: true
---

# Developer Boards supported by Eclipse Oniro

> This section goes into the details of the Eclipse Oniro supported developer
> boards. Supported here means in active use by its developers and parts of the
> CI/CD pipeline guarding the development.

Reviewer: ```suggestion
This section delves into the specifics of Eclipse Oniro's supported developer 
boards. 'Supported' here means being in active use by its developers and being part of the 
CI/CD pipeline that guards the development processes.
```

> Currently this only cover the [HiHope SCDAY200 development board](hh-scdayu200.html).

Reviewer: ```suggestion
Currently this only cover the [HiHope SCDAYU200 development board](hh-scdayu200.html).
```
